### ISSUE TITLE
magic mouse 2 right click does not work well when run 3.1

### ISSUE THREAD

**Issue by lujimmy** [MacGesture/MacGesture#117]
> 🇬🇧 Please write the feedback in English if possible so everyone who might read it will be able to respond easily! 👍

**Describe the bug**
magic mouse 2 right click do not work well when run Macgesture 3.1 and above version

**Steps To Reproduce**
Steps to reproduce the behaviour:
1. enable Macgesture 3.1
2. right click in Safari or any other apps
3. right click with magic mouse 2 does not work every click
4. right click with touch pad works well
5. disable Macgesture 3.1
6. right click with magic mouse 2 works well
7. try with other version, this bug remained

**Expected behaviour**
right click with magic mouse 2 works well

**Desktop (please complete the following information):**
 - OS:macOS Monterey 12.3 (21E230)
 - Browser:safari 15.4 (17613.1.17.1.6)

**Additional context**
optimal the right click logic with time or move space, under a limited define right click else a gesture.

BTW, Macgesture is wonderful thx for your hard work

**Comment by Shavcat** [comment]
I confirm this issue. Definitely there needed a cursor move limit so gesture doesn't start unless let's say under 10 pixels move.

**Comment by username0x0a** [comment]
There's a detection threshold value already available, yet it's not exposed to the UI at this moment. 🤔 But you can fortunately experiment with it using User Defaults via Terminal:

```
defaults write com.codefalling.MacGesture directionDetectionThreshold -int 20
```

where `20` is the default value, so the gestures aren't detected & applied if you move the cursor less than 20 pixels in any recognisable direction.

**Comment by Shavcat** [comment]
Yeah but the app still draws the path and prevents right click menu to pop out. The only solution for now is to try keep your cursor not move when you press right click button

**Comment by username0x0a** [comment]
Well, experimenting with it just now 🤔 can you try turning the Mouse path off? 😄 Seems like it behaves differently with this setting specifically (Path on → below threshold events are blocked) so it would hopefully just be a nasty, but easy-to-fix bug.

I don't really use any indication so that's probably worked fine on my side. #lucky 😄

**Comment by Shavcat** [comment]
Wow, now works just fine after turning the path off 😁🙏

**Comment by username0x0a** [comment]
Okaaay, so that'd be it 😄✅ it should work that way even if the Mouse path is shown, I'll try to address it soon. 👌 Keeping the Mouse path off in the meantime is the workaround until then.

**Comment by bytemain** [comment]
Please try https://github.com/MacGesture/MacGesture/releases/tag/3.2.0

I think it fixed problem metioned here.
